.github/workflows/auto-update-pr-targeting-release.yml

```diff
@@ -0,0 +1,76 @@
+name: Auto-update for PR targeting release branch
+
+on:
+  pull_request:
+    branches:
+      - Version-v*
+
+jobs:
+  auto-update:
+    name: Auto-update
+    runs-on: ubuntu-latest
+    env:
+      YARN_ENABLE_IMMUTABLE_INSTALLS: false
```

Comment:
Why would we disable immutable installs? We don't want the bot updating dependencies for us. Those require review.

Reply:
In the specific instance of a cherry-picked PR, it frequently has conflicts in `yarn.lock`, and this allows the bot to solve certain types of `yarn.lock` merge conflicts all by itself.  You'll still review what the bot did, but the Release Engineer doesn't have to manually do the merge.

Reply:
Makes sense, sorry I thought this was on the RC itself.